Source/KSCrash/Recording/Monitors/KSCrashMonitor_CPPException.cpp

```diff
@@ -184,13 +184,25 @@ catch(TYPE value)\
 #pragma mark - Public API -
 // ============================================================================
 
+static void initialize()
+{
+    static bool isInitialized = false;
+    if(!isInitialized)
+    {
+        isInitialized = true;
+        kssc_initCursor(&g_stackCursor, NULL, NULL);
```

Comment:
By the time terminate is called, the stack trace has been obliterated by the C++ runtime, so it has to be initialized before an exception occurs.